### PR TITLE
Fixed sorting of structure views

### PR DESCRIPTION
**Pull request by dreamxyp**
Fixed sorting of structure views to allow the user to choose whether to sort alphabetically
修正了结构视图的排序，允许用户选择是否按字母排序

### CODE REVIEW

**Comment by dreamxyp** [comment]
Help me merge the code from Erlide, I'm not used to not having this feature.

**Comment by dreamxyp** [comment]
image

image

**Comment by dreamxyp** [comment]
image

File: .idea/dictionaries/Administrator.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<component name="ProjectDictionaryState">
+  <dictionary name="Administrator">
+    <words>
+      <w>darcula</w>
+      <w>dreamxyp</w>
```
Comment (kvakvs) [inline .idea/dictionaries/Administrator.xml:5]: What does that do?

**Comment by kvakvs** [comment]
I see many code changes not related to the feature (IDEA XML files, README, etc)

**Comment by kvakvs** [comment]
Only modify files which contain your changes
Do not change other project files
Do not change copyright and author name

**Comment by dreamxyp** [comment]
Ok, I will try to submit it again after correction

**Comment by kvakvs** [comment]
Tested this and i like it
Thanks for contribution

**Pull request merged**